user/sms-configuration.md

```diff
@@ -0,0 +1,298 @@
+<!--DHIS2-SECTION-ID:sms-configuration-commands-edit-keyvalue-->
+
+This command type works identically to J2ME parser commands work. Please check
+the documentation above.
+
+#### Editing program stage data entry parser commands
+<!--DHIS2-SECTION-ID:sms-configuration-commands-edit-programstagedataentry-->
+
+Commands of this parser type have all the shared fields.  Additionally short
+codes can be defined. A short code can be added for every data element
+connected to the program stage that was selected when adding the command:
+
+![SMS Commands - Program stage data entry parser short codes](resources/images/sms-configuration/commands-edit-programstagedataentryparser-shortcodes.png)
+
+#### Editing tracked entity registration parser commands
+<!--DHIS2-SECTION-ID:sms-configuration-commands-edit-trackedentityregistration-->
+
+Commands of this parser type have all the shared fields.  Additionally short
+codes can be defined. A short code can be added for every tracked entity
+attribute connected to the program that was selected when adding the command:
+
+![SMS Commands - Program stage data entry parser short codes](resources/images/sms-configuration/commands-edit-trackedentityregistrationparser-shortcodes.png)
+
+#### Editing unregistered parser commands
+<!--DHIS2-SECTION-ID:sms-configuration-commands-edit-unregistered-->
+
+This command type works identically to alert parser commands work. Please check
+the documentation above.
+
+## SMS Gateways
+<!--DHIS2-SECTION-ID:sms-configuration-gateways-->
+
+An SMS gateway lets a DHIS2 instance send and receive SMS messages. Different
+gateway types can be added and configured below. At least one gateway is needed
+to send and receive SMS messages. Load balancing will use all gateways if there
+are multiple available.
+
+There are four types of Gateways supported by the SMS Service:
+* Generic gateways
+* BulkSMS gateways
+* Clickatell gateways
+* SMPP gateways
+
+### Listing gateways
+<!--DHIS2-SECTION-ID:sms-configuration-gateways-listing-->
+
+To see all available gateways, navigate to the "Gateway configurations" page.
+All gateways will be listed in a table. On this page the following actions are
+available:
+
+* Add gateways (1)
+* Edit gateways (2)
+* Delete gateways (3)
+* Batch-delete gateways (4)
+* Set the default gateway (5)
+
+![SMS Gateways - List](resources/images/sms-configuration/gateways-list.png)
+
+### Adding gateways
+<!--DHIS2-SECTION-ID:sms-configuration-gateways-add-->
+
+After clicking on the "Add gateway" button in the gateway configurations
+overview page, a dynamic form will appear. It's devided into two sections:
+
+1. The gateway type (1)
+1. The gateway's configuration, depends on the gateway type (2)
+
+![SMS Gateways - Adding gateways](resources/images/sms-configuration/gateways-addinggateways.png)
+
+For more details about configuring gateways, please refer to the [Gateway
+Configurations](#gateway.configuration) section of the "Mobile" maintenance
+documentation.
```

Comment:
@Philip-Larsen-Donnelly I'm linking to another document here. Is the link url / hash-tag correct? Or do I need to add sth else to link to another document?